INVENTORY.md

```diff
@@ -106,3 +108,4 @@ and [Spectrum Web Components](https://opensource.adobe.com/spectrum-web-componen
 | typography          |                    | [📄](https://opensource.adobe.com/spectrum-css/typography.html)         | [📄](https://opensource.adobe.com/spectrum-web-components/components/styles)              |
 | underlay            |                    |                                                                         | [📄](https://opensource.adobe.com/spectrum-web-components/components/underlay)            |
 | well                | ✅                 | [📄](https://opensource.adobe.com/spectrum-css/well.html)               | ❌                                                                                        |
+| undefined           |                    |                                                                         | [📄](https://opensource.adobe.com/spectrum-web-components/components/swatch)              |
```

Comment:
This looks like the script breaks here?

Reply:
If it feels easy to address, let's do it. If not, the update is still better than the out of date version.

Reply:
🤦